uijson/export_to_las.ui.json

```diff
@@ -0,0 +1,28 @@
+{
+    "title": "Export drillhole group",
+    "geoh5": "",
+    "run_command": "las_geoh5.main",
+    "run_command_boolean": {
+        "value": false,
+        "label": "Run python module ",
+        "tooltip": "Warning: launches process to run python model on save",
+        "main": true
+    },
+    "monitoring_directory": "",
+    "conda_environment": "las-geoh5",

```

Comment:
**suggestion:** will be part of the mirageo installation
```suggestion
    "conda_environment": "mirageo",

```

Reply:
Maybe, but this package should allow to run by itself?

Reply:
done.